app/infrastructure/db/schema.ts

```diff
@@ -0,0 +1,69 @@
+import { sql } from "drizzle-orm"
+import {
+  check,
+  index,
+  integer,
+  pgTable,
+  primaryKey,
+  serial,
+  text,
+} from "drizzle-orm/pg-core"
+
+/** 商品 */
+export const productTable = pgTable(
+  "product",
+  {
+    id: serial("id").primaryKey(),
```

Comment:
nits: 一応`SERIAL`の代わりに`GENERATED AS IDENTITY`を使っておいた方が、意図しない値の手動指定を防げて良い気がします？


```suggestion
    id: integer("id").primaryKey().generatedAlwaysAsIdentity(),
```

cf.

https://future-architect.github.io/articles/20241113a/